packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -15,30 +15,35 @@ interface ModalProps {
 }
 
 const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   text-align: center;
   overflow-x: hidden;
-  padding: 2em;
```

Comment:
I changed these to rems as I found that the ems were a bit hard to maintain here since the font size in context could be coming from a few different places, the rems still fulfills the same accessibility requirement.